src/openms/source/ANALYSIS/ID/FIAMSDataProcessor.cpp

```diff
@@ -146,20 +146,21 @@ namespace OpenMS {
     return output;
   }
 
-  void FIAMSDataProcessor::runAccurateMassSearch(FeatureMap& input, OpenMS::MzTab& output, OpenMS::MzTabM& output2) {
+  void FIAMSDataProcessor::runAccurateMassSearch(FeatureMap& input, OpenMS::MzTab& output) {
```

Comment:
```suggestion
  void FIAMSDataProcessor::runAccurateMassSearch(FeatureMap& input, OpenMS::MzTab& output)
  {
```

Reply:
The whole class (FIAMSDataProcessor) seems to follow the structure
```
function() {
}
```